src/tt21100.toit

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the package's LICENSE file.
+
+import gpio
+import i2c
+
+TT2100_FLIP_X  := 0x1
+TT2100_FLIP_Y  := 0x2
+TT2100_FLIP_XY := 0x4
+
+INVALID_VALUE_ ::= -1
+
+bytes2ushort bytes/ByteArray -> int:
```

Comment:
I would probably use the `LITTLE_ENDIAN` class from `import binary` instead.
See below (the comment on `bytes2ushort data[0..2]`).